File: java/8kyu/AreaOfASquare.java
/*

Author: MARCEL EBALLAR
GitHub: https://github.com/MACLEB1903

Source:
https://www.codewars.com/kata/5748838ce2fab90b86001b1a/train/java


Instructions:
Complete the function that calculates the area of the red square, when the length of the circular arc A is given as the input. Return the result rounded to two decimals.

Note: use the π value provided in your language (Math::PI, M_PI, math.pi, etc)

*/

public class Geometry{
  public static double squareArea(double A){
        double l = (2*A)/Math.PI;
        double x = l*l;
        double y = Math.round(x * 100) * 0.01d;
        return y;
    }
}

File: typescript/8kyu/area-of-a-square.ts
/*

Author: MARCEL EBALLAR
GitHub: https://github.com/MACLEB1903

Source:
https://www.codewars.com/kata/5748838ce2fab90b86001b1a/train/typescript


Instructions:
Write the function squareArea(A) (square_area($A) in PHP) that finds the area of the red square when you have the length of the circular arc A.

Use π = Math.PI (M_PI in PHP)
Round to two decimals.

*/

function squareArea(A: number): number {
  return Number(Math.pow((2 * A) / 3.1416, 2).toFixed(2));
}
